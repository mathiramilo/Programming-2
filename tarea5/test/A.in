hacerVecinos 1 2 1.0
sonVecinos 1 2
sonVecinos 3 4
Fin
