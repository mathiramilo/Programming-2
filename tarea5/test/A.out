1>Nuevos vecinos: 1 2 1.00.
2>Son vecinos.
3>NO son vecinos.
4>Fin.
